crates/bootstrap_srv/Makefile.toml

```diff
@@ -1 +1,15 @@
 extend = "../../Makefile.toml"
+
+[tasks.clippy]
+args = [
+  "clippy",
+  "--all-targets",
+  "--no-default-features",
+  "--features",
+  "iroh-relay",
+  "--",
+  "--deny=warnings",
+]
```

Comment:
Only lint with Iroh? Seems like this should maybe be an extra task like we have with go-pion vs datachannel?